tests/testthat/dummy_packages/assignmentLinter/tests/testthat.R

```diff
@@ -0,0 +1,12 @@
+# This file is part of the standard setup for testthat.
+# It is recommended that you do not modify it.
+#
+# Where should you do additional test configuration?
+# Learn more about the roles of various files in:
+# * https://r-pkgs.org/tests.html
+# * https://testthat.r-lib.org/reference/test_package.html#special-files
+
```

Comment:
Nit: We don't need to include this. It is unnecessarily distracting.

Reply:
removed. We can always recreate this if we want to.

Reply:
Sorry for the misunderstanding. 

I meant that we don't need that comment in the `testthat.R` file. It would still be good to have the `testthat.R` file without those comments. Otherwise, it'd be confusing to have tests without nothing executing those tests.